packages/backend/src/dbt/translator.test.ts

```diff
@@ -140,11 +136,11 @@ describe('convert tables from dbt models', () => {
         expect(
             convertTable(
                 SupportedDbtAdapter.SNOWFLAKE,
-                MODEL_WITH_NO_TIME_INTERVAL_DIMENSIONS_SNOWFLAKE,
+                MODEL_WITH_NO_TIME_INTERVAL_DIMENSIONS,
```

Comment:
Surely these two Snowflake tests will fail because the syntax is in reverse to bigquery?

Reply:
and also as it is datetime_trunc and not date_trunc on bigquery

Reply:
how are we deciding the test coverage? I thought because there was two possible different types of syntax that we needed two sets of tests.

Reply:
The model is always the same, what changes is the expected result.

Reply:
ah ok makes sense

Reply:
> how are we deciding the test coverage? I thought because there was two possible different types of syntax that we needed two sets of tests.

Right now we don't have a strict testing coverage strategy. We are slowly adding tests when we see fit and have time. Saying that, I know we could write way more tests for each column type and warehouse but I don't find it critical right now.

Reply:
I took a look at the files in the IDE again with your changes, and I understand how you're reusing the results for bq default and snowflake default for no intervals set. All make's sense now thanks.